xbmc/ServiceManager.h

```diff
@@ -72,6 +72,7 @@ class CServiceManager
   CPlatform& GetPlatform();
 
   PLAYLIST::CPlayListPlayer& GetPlaylistPlayer();
+  int init_level = 0;
```

Comment:
oh, and the issue already exists in mainline, it just doesn't bite that often (you need to have a file type handled by a audio decoder with multitrack support sitting in some directory listed at early stages in the application). it bit me with rar cause i had some archive in a temp dir which made me notice..